### DISCUSSION TITLE
Quill Editor doesn't add br to code block

### DISCUSSION THREAD

**Discussion by ferdisahin** [orchidsoftware/platform#1845] [Q&A]
I add a code block with the Quill editor, but the br tag is not added to the codes I add. How can I solve this? Thanks. 

Quill Editor code block
![Ekran Resmi 2021-08-21 18 08 47](https://user-images.githubusercontent.com/12788397/130326239-cfdebb18-b785-4597-8a06-aeb3b0adffd7.png)

HTML
![Ekran Resmi 2021-08-21 18 08 59](https://user-images.githubusercontent.com/12788397/130326251-f78e342e-90fa-4491-95a3-844781ae22e2.png)

Frontend Page
![Ekran Resmi 2021-08-21 18 09 14](https://user-images.githubusercontent.com/12788397/130326255-6f049084-e21e-43c9-966c-364dafc37547.png)

**Comment by falconandrea** [comment]
I also have the same problem, no news?
